src/Features/VisualBasic/Portable/Diagnostics/Analyzers/VisualBasicPreferFrameworkTypeDiagnosticAnalyzer.vb

```diff
@@ -0,0 +1,60 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Diagnostics.PreferFrameworkType
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Diagnostics.Analyzers
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Friend Class VisualBasicPreferFrameworkTypeDiagnosticAnalyzer
+        Inherits PreferFrameworkTypeDiagnosticAnalyzerBase(Of SyntaxKind)
+
+        Protected Overrides ReadOnly Property SyntaxKindsOfInterest As ImmutableArray(Of SyntaxKind)
+            Get
+                Return ImmutableArray.Create(SyntaxKind.PredefinedType)
+            End Get
+        End Property
+
+        Protected Overrides Function IsInMemberAccessContext(node As SyntaxNode, semanticModel As SemanticModel) As Boolean
```

Comment:
If we're introducing an extension that literally only checks if something is in a MemberAccess context, then this method shouldn't have the same name because it doesn't have the same semantics.